braille-printer-app/NOTICE

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
Please update the copyright/author name part of `NOTICE` in the main directory. Do not add an extra `NOTICE` file here.